tests/lib/rules/jsx-no-target-blank.js

```diff
@@ -96,60 +96,97 @@ ruleTester.run('jsx-no-target-blank', rule, {
   ],
   invalid: [{
     code: '<a target="_blank" href="http://example.com"></a>',
+    output: '<a target="_blank" rel="noreferrer" href="http://example.com"></a>',
     errors: defaultErrors
   }, {
     code: '<a target="_blank" rel="" href="http://example.com"></a>',
+    output: '<a target="_blank" rel="noreferrer" href="http://example.com"></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" rel={0} href="http://example.com"></a>',
+    output: '<a target="_blank" rel={"noreferrer"} href="http://example.com"></a>',
+    errors: defaultErrors
+  }, {
+    code: '<a target="_blank" rel={false} href="http://example.com"></a>',
+    output: '<a target="_blank" rel={"noreferrer"} href="http://example.com"></a>',
```

Comment:
```suggestion
    output: '<a target="_blank" rel="noreferrer" href="http://example.com"></a>',
```
we shouldn't include the curly braces for a string literal autofix.